app/assets/javascripts/miq_explorer.js

```diff
@@ -0,0 +1,167 @@
+ManageIQ.explorer = {}
+
+ManageIQ.explorer.updateElement = function(element, options) {
+  if ( _.isString(options.legend) ) 
```

Comment:
I think @romanblanco converted all the ifs to use braces at some point, even one-liners..